modules/lb-vshn-hiera/main.tf

```
@@ -0,0 +1,22 @@
+locals {
+  instance_fqdns = formatlist("%s.${var.node_name_suffix}", var.lb_names)
+
+  lb_count = length(var.lb_names)
+
+  content = templatefile(
+    "${path.module}/templates/hieradata.yaml.tmpl",
+    {
+      "cluster_id"   = var.cluster_id
+      "api_vip"      = cidrhost(var.api_vip_network, 0)
+      "router_vip"   = cidrhost(var.router_vip_network, 0)
```

The cidrhost logic is only required for cloudscale.ch, because the "cloudscale_floating_ip" resource only gives the cidr notation for the floating IP, remove it here, and do it when we instantiate this module for the cloudscale.ch LBs.

```suggestion
      "api_vip"      = var.api_vip
      "router_vip"   = var.router_vip
```